documentation/asset-tracker/topics/assettracker-hardware.md

```diff
@@ -0,0 +1,194 @@
+:alt: AssetTracker deployment diagram 3
+:class: border-black
+
+Example Daisy Chain Deployment
+```
+
+For installing AssetTracker in multiple racks in a row, you may daisy-chain up to five Expansion Hubs to a single gateway. The maximum length of the cables from the Gateway to the last Expansion Hub in the chain must not exceed _20 meters (65 feet)_.
+
+Each Master Module has an address between 1 (one) and 5 (five). Each Master Module must have a _unique_ address in the Daisy Chain configuration. The address specification is printed on the Master Module tag.
+
+```{figure} /asset-tracker/media/master_modules_1.png
+:alt: Master Modules image 1
+:class: border-black
+
+Master Modules with ID and address (Circled)
+```
+
+### Network requirements
+
+:::{important}
+You must [contact our support team](https://system.hyperviewhq.com/helpdesk) to get access to the latest vendor configuration software and documentation.
+:::
+
+- The Gateway uses port 5656/TCP for management and configuration. The configuration must be done using the supplied tool. It is strongly recommended that the tool be one the same network segment as the gateway to minimize network related issues. A copy of the configuration tool will be provided as part of the license activation and onboarding process.
+
+- The Gateway communicates with the data collector over port 1883/TCP. Communication is from the gateway to the Data Collector. The {ref}`Data Collector setup documentation<Setting-up-data-collectors-doc>` has more information on the network requirements.
+
+### AssetTracker Strip sizing
+
+AssetTracker modules come in a 5U and 6U modular design that can be connected for configuration on any rack RU size. Based on the rack size, use a combination of the 5U and 6U modules to fit exactly, or close to the rack height.
+
+### Installation locations
+
+Typical installation locations include but are not limited to:
+
+- Rack side panel
+- Rack door
+- Rack front panel
+
+In all these cases please:
+
+- Verify there is enough space for the modules and 3M adhesive.
+- Verify there is enough space to mount and remove equipment from the rack.
+- Verify the door can be opened and closed without interference.
+- Verify that there is not interference with power, network and other infrastructure.
+- Verify that the AssetTracker Strip is supported on a flat surface with no mechanical stress or pressure on it.
+
+Use the adhesive where possible and if needed use plastic zip ties for re-enforcement. Take care not to add pressure, puncture the casing or create mechanical stress on the assembled device.
+
+```{figure} /asset-tracker/media/installation_example_1.png
+:alt: Master Module and SATA cable
+:class: border-black
+
+Rack installation example
+```
+
+### Assembling Modules
+
+Identify the Master Module. The Master module has a SATA port above the top-most magnet position. The Extension Modules have male connectors on the top-end.
```

Comment:
There are some inconsistencies in the document when referencing "Master Module". Sometimes it's "Master Module" and sometimes it's "Master module" and sometimes it's "master module".

Reply:
Same with "Extension Modules"

Reply:
and "AssetTracker Strip"

Reply:
and "Expansion Hub"

Reply:
and "Gateway"

Reply:
and "Asset Tags"

Reply:
Fixed, Thanks!